playground/infrastructure/helm-playground/values.yaml

```diff
@@ -29,4 +29,12 @@ autoscaling:
    enabled: false
    minReplicas: 1
    maxReplicas: 4
-   targetCPUUtilizationPercentage: 80
\ No newline at end of file
+   targetCPUUtilizationPercentage: 80
+static_ip: 34.120.95.76
+redis_ip: 10.226.29.100:6379
+project_id: astst-369706
+registry: us-east1-docker.pkg.dev/astst-369706/playground-repository
+static_ip_name: pg-static-ip
+tag: teta
+dns_name: dev-playground.online
```

Comment:
Do we want to hardcode these?

Reply:
Those variables are changing automatically thru the Gradle task (if they exist) or will be created during execution. Removed for now